addons/survey/models/survey_user_input.py

```diff
@@ -623,6 +625,24 @@ def _message_get_suggested_recipients(self):
                 )
         return recipients
 
+    def _notify_new_participation_subscribers(self):
+        partner_root_id = self.env.ref('base.partner_root').id
+        subtype_id = self.env.ref('survey.mt_survey_survey_user_input_completed', raise_if_not_found=False)
+        if not self.ids or not subtype_id:
+            return
+        recipients_data = self.env['mail.followers']._get_recipient_data(self.survey_id, 'notification', subtype_id.id)
+        followed_survey_ids = [survey_id for survey_id, followers in recipients_data.items() if followers]
+        for user_input in self.filtered(lambda user_input: user_input.survey_id.id in followed_survey_ids):
+            user_input.message_post(
+                author_id=partner_root_id,
```

Comment:
Why ?

Reply:
Mmmh ... 'author_id=self.env.user.partner_id.id if not self.env.user.is_public() else partner_root_id' ?

Reply:
So in the case the author is identified, instead of 
`OdooBot: Raoul just participated` you'd prefer `Raoul: I just participated` while keeping `OdooBot: Someone just participated` otherwise?

Reply:
In that case, we're back to using ugly public user otherwise it doesn't make sense ^^